st2common/st2common/services/action.py

```diff
@@ -98,6 +98,11 @@ def create_request(liveaction):
     liveaction.status = action_constants.LIVEACTION_STATUS_REQUESTED
     liveaction.start_timestamp = date_utils.get_datetime_utc_now()
 
+    liveaction.log = [{
```

Comment:
I'd prefer a util method that either adds to array or appends it. The current way is prone to errors (programmer).